src/Internal/ClusterDnsEndPointDiscoverer.php

```diff
@@ -258,7 +258,7 @@ private function tryGetGossipFrom(GossipSeed $endPoint): Promise
                 return null;
             }
 
-            $json = yield $response->getBody()->getInputStream()->read();
+            $json = yield $response->getBody()->read();
```

Comment:
@prolic Can you take a look at this and the `$deferred->resolve($response->getBody()->buffer());` calls bellow? I'm not sure when to use `read` and when to use `buffer`.

Reply:
`buffer()` reads everything at once while `read()` only reads a chunk.